data/health/health-small-statistical-areas/index.html

```diff
@@ -5,45 +5,58 @@
   display_name: JP
   email: agrc@utah.gov
 date: 2011-11-21 17:25:17 -0700
-title: Small Health Statistical Areas
 categories: []
+2020SmallHealthStatisticalAreas:
```

Comment:
**thought**: i wonder if we should only keep the 2020 as a main level with the prior years as additional information. The metadata would then only be managed in arcgis online. what do you think?

Reply:
yeah I am not too excited about hammering into someone else's pages of data I know nothing about. I think we should merge what I have and revisit the page later, I know it needs more work.